runtime/frequency/src/lib.rs

```diff
@@ -264,9 +264,11 @@ impl Contains<RuntimeCall> for BaseCallFilter {
 			match call {
 				RuntimeCall::Utility(pallet_utility_call) =>
 					Self::is_utility_call_allowed(pallet_utility_call),
-				// Create provider and create schema are not allowed in mainnet for now. See propose functions.
+				// Create provider and create schema/intent/intent group are not allowed in mainnet for now. See propose functions.
 				RuntimeCall::Msa(pallet_msa::Call::create_provider { .. }) => false,
 				RuntimeCall::Schemas(pallet_schemas::Call::create_schema_v3 { .. }) => false,
+				RuntimeCall::Schemas(pallet_schemas::Call::create_intent { .. }) => false,
```

Comment:
you should add these inside
`fn is_batch_call_allowed(call: &RuntimeCall) -> bool {`

Reply:
We should also add `update_intent_group`?